spec/pb_kits/playbook/kits/icon_circle_spec.rb

```diff
@@ -17,11 +17,17 @@
 
   describe "#classname" do
     it "returns namespaced class name", :aggregate_failures do
-      expect(subject.new({}).classname).to eq "pb_icon_circle_kit_md_default"
-      expect(subject.new(variant: "royal").classname).to eq "pb_icon_circle_kit_md_royal"
-      expect(subject.new(size: "sm").classname).to eq "pb_icon_circle_kit_sm_default"
-      expect(subject.new(variant: "purple", size: "lg").classname).to eq "pb_icon_circle_kit_lg_purple"
-      expect(subject.new(classname: "additional_class").classname).to eq "pb_icon_circle_kit_md_default additional_class"
+      icon = "user"
+      expect(subject.new(icon: icon, variant: "royal").classname).to eq "pb_icon_circle_kit_md_royal"
+      expect(subject.new(icon: icon, size: "sm").classname).to eq "pb_icon_circle_kit_sm_default"
+      expect(subject.new(icon: icon, variant: "purple", size: "lg").classname).to eq "pb_icon_circle_kit_lg_purple"
+      expect(subject.new(icon: icon, classname: "additional_class").classname).to eq "pb_icon_circle_kit_md_default additional_class"
+    end
+
+    it "raises an error when not given an icon" do
+      expect { subject.new({}) }.to raise_error(Playbook::Props::Error)
```

Comment:
Nice!

Reply:
This seems to be a duplicated test of what happens on line 10